src/build-scripts/serialize-bc.py

```diff
@@ -1,3 +1,4 @@
+#!/bin/env python
```

Comment:
Isn't this traditionally `/usr/bin/env` ?

Reply:
Correct, this is more in line with the other builds scripts in the project.